python/Ganga/GPIDev/Lib/Job/Job.py

```diff
@@ -2091,20 +2089,19 @@ def __setattr__(self, attr, value):
 
             # reduce duplicate values here, leave only duplicates for LCG,
             # where we can have replicas
-            uniqueValuesDict = []
-            uniqueValues = []
+            uniqueValuesDict = oDict()
```

Comment:
Possibly, but this equality check was not being performed until now, all I did was improve what was here but not completely rewrite it.
If we're of the mindset that we should opt for this approach then great, a load of extra noise here reduces away very quickly and I become a lot happier.